pkg/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.nupkgproj

```diff
@@ -7,8 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-    <!-- This is only needed until https://github.com/dotnet/corefx/issues/31064 is addressed. -->
-    <PackageReference Include="System.Runtime.Intrinsics.Experimental" Version="4.6.0-preview1-26708-04" />
+   <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
```

Comment:
This package reference needs to be included unconditionally. Here you are only adding it when `TargetFramework` == `netcoreapp3.0`.

Reply:
@eerhardt ok got it , so it should be available for all .NET Core versions right ?

Reply:
Yes, correct. We use `Span<T>` for all versions of this package/library.

Reply:
@eerhardt So in this case I have removed the condition, is there anything else required for this? Sorry I didn't get the usage of `Span<T>` in this scenario.

Reply:
> is there anything else required for this?

No, I think this should do it.

> Sorry I didn't get the usage of Span<T> in this scenario.

`System.Memory` is the package that defines the `Span<T>` type, which is what we are using in our code. Thus causing the dependency on the `System.Memory` package.

Reply:
Actually, I take the `this should do it` back.

Looking at the `.csproj`, we only use that `PackageReference` on `netstandard2.0`:

https://github.com/dotnet/machinelearning/blob/00021b69f3ab56ee6c78d86aca5728ec971181b8/src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj#L27-L33

I think we can probably skip adding the package reference on `netcoreapp3.0`, since `Span<T>` is "in-box" on that version.

/cc @ericstj - thoughts?

Reply:
@eerhardt ok , so you mean as it's referring to netstandard2.0 only in the project file, then we need to keep that condition in the nupkgproj file. Am I right?

Reply:
@eerhardt  ok , I have made the changes, does it look fine now?

Reply:
Looks perfect! Thanks!